gradoop-store/gradoop-hbase/src/main/java/org/gradoop/storage/impl/hbase/HBaseEPGMStore.java

```diff
@@ -57,7 +57,7 @@
  */
 public class HBaseEPGMStore implements
```

Comment:
Duplicate code ::getGraphSpace, ::getVertexSpace and ::getEdgeSpace. Consider using a private function instead

Reply:
Maybe you can give me a hint how to do that. The problem is the generics of `HBaseElementFilter<>`

Reply:
using a function generic like 

```java
  private <T extends EPGMElement> void attachFilter(
    @Nonnull ElementQuery<HBaseElementFilter<T>> query,
    @Nonnull Scan scan
  ) {
    //...
  }
```

Reply:
Thx a lot